src/runtimes/worker/auth/fetch_auth.ts

```diff
@@ -28,10 +29,10 @@ var fetchAuth: AuthTransport = function(
       let { status } = response;
       if (status === 200) {
         return response.text();
-      } else {
-        Logger.error("Couldn't get auth info from your auth endpoint", status);
-        throw status;
       }
+      let message = `Couldn't get auth info from your auth endpoint, status: ${status}`;
```

Comment:
Nit pick this might be a little clearer for some people
```suggestion
      let message = `Could not get auth info from your auth endpoint, status: ${status}`;
```